ganymede/models/stahl/cd_locate_stores.sql

```diff
@@ -0,0 +1,9 @@
+{{ config(materialized='view') }}
+
+with store_locations as (
+    select id, r_store_attributes->'geo'->'latitude' as latitude, r_store_attributes->'geo'->'longitude' as longitude, {{ raw_region_to_province("store_address->>'addressRegion'") }} as region
+    from {{ source('aethervest', 'stores') }}
+)
+select store_locations.id, cds.cdname, store_locations.region 
+from store_locations inner join {{ source('static_resources', 'censusdivisions') }} cds
+on ST_Covers(cds.geom, ST_Transform(ST_SetSRID(ST_MakePoint(longitude::double precision, latitude::double precision), 4326), 3347))
```

Comment:
Can you use words to describe to me what's happening with this function here? New to me as well.

Reply:
I don't really follow the these `srid` integer values... scanned the docs and I'm guessing they pertain to some coordinate system ::shrugs::?